Cargo.toml

```diff
@@ -39,6 +39,3 @@ rand = { version = "0.7", features = ["small_rng"] }
 regex = "1.3"
 scopeguard = "1.0"
 tempdir = "0.3"
-
-[build-dependencies]
-hdf5-sys = { path = "hdf5-sys", version = "0.6.1" }  # !V
```

Comment:
@magnusuMET So it's no longer a build dependency?

Reply:
Yes, we only need to build this library once, not also during build phase. build scripts are built as debug, so this is really necessary when compiling also `hdf5-src`!